templates/SilverStripe/Admin/Includes/LeftAndMain_MenuToggle.ss

```diff
@@ -4,15 +4,10 @@
         <span class="cms-help__logo font-icon-silverstripe"></span>
```

Comment:
This template has zero value if you don't insert the CWP version in here ... just delete.

Reply:
Deleted